lib/Utilities/src/Logging.cpp

```diff
@@ -114,14 +114,17 @@ bool Logging::selectSink(const String& name)
 
     while ((MAX_SINKS > index) && (false == status))
     {
-        if (m_sinks[index]->getName() == name)
+        if (nullptr != m_sinks[index])

```

Comment:
In case there is no sink registered, the index will not be updated and its an infinite loop.
Unfortuantely I saw this just one second later after merge. Thats why I reverted it again. Please fix and request again.